bundles/org.openhab.binding.pilight/src/main/resources/OH-INF/thing/devices.xml

```diff
@@ -67,6 +67,10 @@
 		<item-type>Contact</item-type>
 		<label>State of Contact</label>
 		<description>State of Pilight Contact.</description>
+		<tags>
+			<tag>Status</tag>
+			<tag>Light</tag>
```

Comment:
The tag `light` does not seem correct here.

Reply:
I will remove the tag as it seems unclear what its meaning should be